### PR TITLE
Update Flex.mdx

### PR DESCRIPTION
**Pull request by Ruyven**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 📝 Test Instructions:

Both cases said `"column"`. Fixed to specify the correct behavior for "row" and match the order of the previous section.

### CODE REVIEW

File: packages/@react-spectrum/layout/docs/Flex.mdx [hunk 1]
```diff
@@ -136,7 +136,7 @@ an `alignItems` value to account for this.
 ### Justification
 
 The `justifyContent` prop can be used to align items along the main axis. When `direction` is `"column"`, this refers
-to vertical alignment, and when `direction` is `"column"` it refers to horizontal alignment.
+to horizontal alignment, and when `direction` is `"row"` it refers to vertical alignment.
```
Comment (snowystinger) [inline packages/@react-spectrum/layout/docs/Flex.mdx:139]:
```suggestion
to vertical alignment, and when `direction` is `"row"` it refers to horizontal alignment.
```

**Review by snowystinger — COMMENTED** [review]
Hey! Thanks for submitting a PR, there was indeed an error there.
See https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Aligning_Items_in_a_Flex_Container for why I made an edit to your fix.
Would you mind signing the CLA? you may need to close and reopen the PR after signing it.  https://react-spectrum.adobe.com/contribute.html#contributor-license-agreement
After you make the code change, the docs will re-run as well, it looks like they just failed due to running out of memory, it happens on occasion.
Thanks again!

File: packages/@react-spectrum/layout/docs/Flex.mdx (hunk 1 above)
Comment (Ruyven) [reply packages/@react-spectrum/layout/docs/Flex.mdx]: Of course, justify-content aligns items on the main axis!
I signed the CLA, and hopefully the docs and tests will all succeed now as well.

**Review by snowystinger — APPROVED** [review]
lgtm!

**Review by LFDanLu — APPROVED** [review]
Thanks for catching this typo!

**Pull request merged**